zcash_client_sqlite/src/wallet/init.rs

```diff
@@ -310,11 +533,365 @@ mod tests {
 
     use super::{init_accounts_table, init_blocks_table, init_wallet_db};
 
+    #[test]
+    fn init_migrate_from_0_3_0() {
+        fn init_0_3_0<P>(
+            wdb: &mut WalletDb<P>,
+            extfvk: &ExtendedFullViewingKey,
+            account: AccountId,
+        ) -> Result<(), rusqlite::Error> {
+            wdb.conn.execute(
+                "CREATE TABLE IF NOT EXISTS accounts (
```

Comment:
```suggestion
                "CREATE TABLE accounts (
```
These don't need to use `IF NOT EXISTS`, and shouldn't because we want to assert that we're creating these tables from scratch.